### PR TITLE
Create API for Email Subscribe

### PR DESCRIPTION
**Pull request by milhamsuryapratama**
> Related to : [this](https://trello.com/c/5VyPENRH)

> Create API for Email Subscribe

> in this task, I already made subscriber api to store the job seeker email. but I don't know, I did it right or not.

### CODE REVIEW

File: app/Http/Controllers/Api/SubscriberController.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+use App\Http\Controllers\Controller;
+use App\Subscriber;
+use App\Tools\ResponseTool;
+use Illuminate\Http\Request;
+
+class SubscriberController extends Controller
+{
+    public function subscribe(Request $request)
+    {
+        $email = $request->email;
```
[the remark this answers is not in the document]
Comment (milhamsuryapratama) [reply app/Http/Controllers/Api/SubscriberController.php:14]: Ok, I see

[the remark this answers is not in the document]
Comment (dns2012) [reply app/Http/Controllers/Api/SubscriberController.php:15]: don't to forget add `phpdoc` each function you created, it use for self contain documentation and following laravel standard

File: app/Services/SubscriberService.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+namespace App\Services;
+
+
+use App\Subscriber;
+use App\Tools\ResponseTool;
+
+class SubscriberService
+{
+    public static function subscribe($email)
```
Comment (dns2012) [inline app/Services/SubscriberService.php:12]: add phpdoc above

File: app/Services/SubscriberService.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        $checkEmail = Subscriber::whereEmail($email)->count();
+        if ($checkEmail == 0) {
```
Comment (dns2012) [inline app/Services/SubscriberService.php:15]: No need to store it as variable, make it through
```suggestion
        if (Subscriber::whereEmail($email)->count()) {
```

File: app/Services/SubscriberService.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+            Subscriber::create([
```
Comment (dns2012) [inline app/Services/SubscriberService.php:16]: if there is only 1 item, make it one line
```suggestion
            Subscriber::create(['email' => $email]);
```

Comment (milhamsuryapratama) [reply app/Services/SubscriberService.php:16]: Ok, I see 😊

**Pull request merged**